blocks/table/table.js

```diff
@@ -32,6 +32,20 @@ function buildMutipleTables(block) {
   mainContent.replaceChildren(...mainTmp.childNodes);
 }
 
+function innerTableHighlightFirstColumn(block, noHead) {
```

Comment:
I am wondering if this can be achieved via pure CSS. As I understand, we need to style the first column differently. Could we use a selector like `td:first-child`

Reply:
@sdmcraft Generally, it's the best and easiest way to reach the goal, but there are some merged cells in our case that I cannot handle by styling. 

<img width="754" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/0916475f-f9a5-4edb-b1a4-e8ca0f9760b7">